src/config/config.js

```diff
@@ -44,21 +44,19 @@ const envVarsSchema = Joi.object({
     AUTH_SERVICE_REFRESH_TOKEN_MULTIPLE_DEVICES: Joi.bool()
         .default(false),
     AUTH_SERVICE_SEED_ADMIN_USERNAME: Joi.string()
-        .alphanum()
-        .min(3)
-        .max(30)
+        .email({ minDomainAtoms: 2 })
         .required()
-        .default('admin')
-        .description('Admin username for seeding only'),
+        .default('admin@example.com')
+        .description('User name of the Seed Admin user. Used in the for seeding process only.'),
```

Comment:
```suggestion
        .description('User name of the Seed Admin user. Used in the seeding process only.'),
```

Reply:
Whoah! The ability to commit suggestions directly. Awesome!